<ui version="4.0" >
 <class>AddFriendWizard</class>
 <widget class="QDialog" name="AddFriendWizard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>192</red>
        <green>192</green>
        <blue>192</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>208</red>
        <green>208</green>
        <blue>208</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>139</red>
        <green>139</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>104</red>
        <green>104</green>
        <blue>104</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="LinkVisited" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Add Friend to my netowork Wizard</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>260</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Next ></string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="textstackedWidget" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>10</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="page" >
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      	<string>This Wizard helps to Connect to your Friends</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pqipemradioButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>311</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>I have a .pqi/.pem file received from a Friend</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="keyradioButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>311</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>I have Key received from a Friend</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="inviteradioButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>120</y>
       <width>321</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>I want to invite a friend to my RetroShare network</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="invitepage" >
    <widget class="QTextEdit" name="emailText" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>190</y>
       <width>401</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text" >
      	<string> Cut and paste the text below into an email.  and send it to all your friends!  When you press next, RetroShare will attempt to launch a Email client </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      	<string>Invite your Friends to join with RetroShare</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="keypage" >
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      	<string>Enter the Key code you got from your friend:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="keyText" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>200</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>When you press next RetroShare will attempt to connect using the key.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pqipempage" >
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Select a pqi/pem file</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLineEdit" name="lineEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="loadfileButton" >
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>90</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string> When you press next RetroShare will attempt to connect using the .pqi/.pem file.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>backButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>keyradioButton</tabstop>
  <tabstop>pqipemradioButton</tabstop>
  <tabstop>inviteradioButton</tabstop>
  <tabstop>keyText</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>loadfileButton</tabstop>
  <tabstop>emailText</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
